code/MSSQLSchemaManager.php

```diff
@@ -395,8 +395,10 @@ protected function alterTableAlterColumn($tableName, $colName, $colSpec)
         // drop *ALL* indexes on a table before proceeding
         // this won't drop primary keys, though
         $indexes = $this->indexNames($tableName);
+        $indexes = array_filter($indexes);
```

Comment:
Removes empty indexes and they can exist when running dev/build on an upgrade path.